bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/header/HeaderImpl.java

```diff
@@ -42,26 +42,19 @@
 
     static final String RESOURCE_TYPE = "core/cif/components/structure/header/v1/header";
     static final String MINICART_NODE_NAME = "minicart";
+    static final String SEARCHBAR_NODE_NAME = "searchbar";
 
     @Inject
     private Page currentPage;
 
     @Inject
     private Resource resource;
 
-    @Inject
-    private String minicartNodeName;
-
-    private Page searchResultsPage;
     private Page navigationRootPage;
 
     @Override
     public String getSearchResultsPageUrl() {
-        if (searchResultsPage == null) {
```

Comment:
Since you disabled the baseline plugin, why not remove the method from the API if we no longer need it?